src/screens/MemberDetail/MemberDetail.tsx

```
@@ -464,16 +560,131 @@
+              className={`d-flex justify-content-between align-items-center py-3 px-4 ${styles.topRadius}`}
+              style={{backgroundColor: "#EAEBEF", color: "#000000"}}
+            >
+              <h3 className="m-0" data-testid="eventsAttended-title">
+                {t('profileDetailHeading')}
+              </h3>
+            </Card.Header>
+            <Card.Body
+              id="tagsAssignedScrollableDiv"
+              data-testid="tagsAssignedScrollableDiv"
+              className={`${styles.cardBody} pe-0`}
+            ></Card.Body>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)